sensor/common/detector/metrics/metrics.go

```diff
@@ -15,13 +16,51 @@ var (
 		Help:      "Time spent in exponential backoff for the ImageScanInternal endpoint",
 		Buckets:   prometheus.ExponentialBuckets(4, 2, 8),
 	})
+	networkPoliciesStored = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "num_network_policies_in_store",
+		Help:      "Number of network policies (per namespace) currently stored in the in-memory store in sensor.",
+	},
+		[]string{
+			// Which namespace the network policy belongs to
+			"k8sNamespace",
+		})
+	networkPoliciesStoreEvents = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "events_network_policy_store_total",
+		Help:      "Events affecting the state of network policy currently stored in the in-memory store in sensor.",
```

Comment:
Nit: See my other comment - `stored in store`

Reply:
Rephrased in  6421fc0eb